pokemongo_bot/cell_workers/evolve_all_worker.py

```diff
@@ -75,13 +77,12 @@ def _should_run(self):
                                .format(lucky_egg_count-1), 'green')
                 else:
                     logger.log('Failed to use lucky egg!', 'red')
-                    skip_evolves = True
-        elif use_lucky_egg: #lucky_egg_count is 0
-            # Skipping evolve so they aren't wasted
-            logger.log('No lucky eggs... skipping evolve!', 'yellow')
-            skip_evolves = True
+                    return False
 
-        return skip_evolves
+        else: # lucky_egg_count is 0
+            # Skipping evolve so pokemon aren't wasted
+            logger.log('No lucky eggs... skipping evolve!', 'yellow')
```

Comment:
I see. Well I'm afraid this fix is not meant to cover that. My intention was only to make batch evolving work again. You can set `"use_lucky_egg": false` to avoid the messages I guess.